qutebrowser/completion/strategies/configmodel.py

```diff
@@ -0,0 +1,194 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2014-2021 Florian Bruhin (The Compiler) <mail@qutebrowser.org>
+#
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <https://www.gnu.org/licenses/>.
+
+"""Functions that return config-related completion models."""
+
+from typing import Optional, Callable, Any, Tuple
+
+from qutebrowser.completion.strategies.strategy import CompletionStrategy
+from qutebrowser.config import configdata, configexc
+from qutebrowser.completion import completionmodel, util
+from qutebrowser.completion.categories import listcategory
+from qutebrowser.commands import runners, cmdexc
+from qutebrowser.keyinput import keyutils
+from qutebrowser.completion.completer import CompletionInfo
+from qutebrowser.completion.completionmodel import CompletionModel
+
+
+class OptionMaker(CompletionStrategy):
+    """A CompletionModel that is generated for several option sets.
+
+    Args:
+        info: The config info that can be passed through.
+        title: The title of the options.
+        predicate: The function for filtering out the options. Takes a single
+                   argument.
+    """
+
+    COLUMN_WIDTHS = (20, 70, 10)
+
+    def __init__(self, title: str, predicate: Optional[Callable[[Optional[Any]], bool]] = None):
```

Comment:
I think the predicate argument can just be `Any` without the `Optional` -- it should never be `None`.